pkg/generators/models/testdata/cases/validation/expected/model_person_test.go

```diff
@@ -0,0 +1,20 @@
+package generatortest
+
+import (
+	"testing"
+
+	validation "github.com/go-ozzo/ozzo-validation/v4"
+	uuid "github.com/satori/go.uuid"
+	"github.com/stretchr/testify/require"
+)
+
+func TestPerson(t *testing.T) {
+	require.NoError(t, validation.Validate(&Person{
+		Age:      120,
+		Gender:   GenderDefault,
+		Name:     "Foobar",
+		Hostname: "localhost",
+		Email:    "foo@localhost",
```

Comment:
this value is not actually valid, please double check it, when i ran it locally I got 

```sh
go test .
--- FAIL: TestPerson (0.00s)
    model_person_test.go:12:
        	Error Trace:	model_person_test.go:12
        	Error:      	Received unexpected error:
        	            	email: must be a valid email address.
        	Test:       	TestPerson
FAIL
FAIL	generatortest	0.335s
FAIL
```

Reply:
I changed the rule to be `EmailFormat` now.